Formula/f/fricas.rb

```diff
@@ -0,0 +1,32 @@
+class Fricas < Formula
+  desc "Advanced computer algebra system"
+  homepage "https://fricas.github.io"
+  url "https://github.com/fricas/fricas/archive/refs/tags/1.3.9.tar.gz"
+  sha256 "c0e498534bf340cd5e1d8b84365991db60ad6ce7a16f4902426ca45ea8cbd715"
+  license "BSD-3-Clause"
+  head "https://github.com/fricas/fricas.git", branch: "master"
+
+  depends_on "libice"
+  depends_on "libsm"
+  depends_on "libx11"
+  depends_on "libxau"
+  depends_on "libxdmcp"
+  depends_on "libxpm"
+  depends_on "libxt"
+  depends_on "sbcl"
+
+  def install
+    Dir.mkdir("build")
+    chdir "build" do
+      ENV.deparallelize
+      system "../configure", *std_configure_args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    assert_match %r{ \(/ \(pi\) 2\)\n},
+      shell_output("echo 'integrate(sqrt(1-x^2),x=-1..1)::InputForm' | #{bin}/fricas -nosman")
```

Comment:
```suggestion
    assert_match %r{ \(/ \(pi\) 2\)\n},
      pipe_output('integrate(sqrt(1-x^2),x=-1..1)::InputForm', bin/"fricas -nosman")
```

Reply:
I have updated the formula following your suggestion.